addons/fix-editor-comments/addon.json

```diff
@@ -38,30 +38,35 @@
     {
       "name": "Fix dragging blocks with comments",
       "id": "fix-drag",
+      "description": "Fixes a bug where if you move a block with a comment attached to it, the comment will appear to move with it initially, but then jump back to where it was before after switching sprites or reloading.",
       "type": "boolean",
       "default": true
     },
     {
       "name": "Add scroll bars to clipped comments",
       "id": "scroll",
+      "description": "Adds scroll bars to long comments and allows you to scroll through them with the scroll wheel.",
       "type": "boolean",
       "default": true
     },
     {
       "name": "Directly edit standalone comments",
```

Comment:
```suggestion
      "name": "Edit standalone comments",
```

Reply:
"What? Be able to edit standalone comments at all?"

Reply:
Yeah that one shouldn't be chnaged.